README.md

```diff
@@ -40,11 +40,76 @@ npm install --save slackapp
 ```
 
 ## Setup
-//TODO
+You can call the SlackApp function with the following options:
```

Comment:
We should either describe what each of these properties is for and means or link down to the API section where it does.  I'm worried about confusion caused by `app_token`, `app_user_id`, `bot_token` and `bot_user_id`.  Do you think we should rename this? Prefix them with `testing_` or `override_` or something similar?